src/core/rtd_database.tf

```diff
@@ -146,3 +146,23 @@ resource "azurerm_cosmosdb_mongo_collection" "file_register" {
   }
 
 }
+
+resource "azurerm_cosmosdb_mongo_collection" "rtd_file_reporter_collection" {
+
+  count = var.enable.rtd.mongodb_storage ? 1 : 0
+
+  name                = "filereports"
+  account_name        = module.cosmosdb_account_mongodb[count.index].name
+  database_name       = azurerm_cosmosdb_mongo_database.rtd_db[count.index].name
+  resource_group_name = azurerm_resource_group.db_rg.name
+
+  index {
+    keys   = ["_id"]
+    unique = true
+  }
+
+  index {
```

Comment:
@and-mora Is `senderCode` unique in this collection?

Reply:
According to current design, yes. I'm going to add the constraint, thank you.